src/output.py

```diff
@@ -0,0 +1,79 @@
+"""
+Notes
+-----
+This module contains the functions for audiobook_gen that take the generated audio tensors and output to audio files,
+as well as assembling the final zip archive for user download.
+"""
+
+__all__ = ['write_audio', 'assemble_zip']
```

Comment:
You shouldn't need this. See other file for more comments.

Reply:
Removed `__all__`.